search-ui/src/resources/corp-info.ts

```diff
@@ -42,12 +42,6 @@ import { CorpClass, CorpTypeCd } from '@/enums'
       corpClass: CorpClass.BC,
       shortDesc: 'BC CCC',
       fullDesc: 'BC Community Contribution Company'
-    }, {
-      corpTypeCd: CorpTypeCd.CCC_CONTINUE_IN,
-      colinInd: true,
-      corpClass: CorpClass.BC,
-      shortDesc: 'CCC CONTINUE IN',
-      fullDesc: 'BC Community Contribution Company'
```

Comment:
this is the only real change. The others are just me reordering to put the continued in versions of each corp type next to the original (since we are treating them as the same in the UI)